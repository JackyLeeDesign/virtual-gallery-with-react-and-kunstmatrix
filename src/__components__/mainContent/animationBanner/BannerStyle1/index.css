.main_banner,
.main_banner_img {
  position: relative;
  width: 100%;
  height: auto;
  z-index: -1;
  overflow: hidden;
}
.boy_1 {
  position: absolute;
  width: 1.4%;
  top: 80%;
  animation: updown 0.8s, LtoR 30s;
  animation-iteration-count: infinite;
  animation-fill-mode: forwards;
  animation-timing-function: ease, linear;
}
.boy_2 {
  position: absolute;
  width: 1.4%;
  left: -10%;
  top: 80%;
  animation: updown 0.9s, LtoR 50s;
  animation-delay: 5s;
  animation-iteration-count: infinite;
  animation-fill-mode: forwards;
  animation-timing-function: ease, linear;
}
.boy_3 {
  position: absolute;
  width: 1.4%;
  margin-left: 100%;
  top: 80%;
  animation: updown 1s, RtoL 60s;
  animation-iteration-count: infinite;
  animation-fill-mode: forwards;
  animation-timing-function: ease, linear;
}
.girl_1 {
  position: absolute;
  width: 1.65%;
  margin-left: 100%;
  top: 80%;
  animation: updown 1.2s, RtoL 80s;
  animation-delay: 5s;
  animation-iteration-count: infinite;
  animation-fill-mode: forwards;
  animation-timing-function: ease, linear;
}
.girl_2 {
  position: absolute;
  width: 1.65%;
  margin-left: 100%;
  top: 80%;
  animation: updown 0.9s, RtoL 50s;
  animation-delay: 7s;
  animation-iteration-count: infinite;
  animation-fill-mode: forwards;
  animation-timing-function: ease, linear;
}
.girl_3 {
  position: absolute;
  width: 1.65%;
  left: -10%;
  top: 80%;
  animation: updown 0.7s, LtoR 35s;
  animation-delay: 10s;
  animation-iteration-count: infinite;
  animation-fill-mode: forwards;
  animation-timing-function: ease, linear;
}
@keyframes updown {
  33% {
    top: 80%;
  }
  66% {
    top: 75%;
  }
  100% {
    top: 80%;
  }
}
@keyframes LtoR {
  from {
    margin-left: 0%;
  }
  to {
    margin-left: 110%;
  }
}
@keyframes RtoL {
  from {
    margin-left: 100%;
  }
  to {
    margin-left: 0%;
  }
}
