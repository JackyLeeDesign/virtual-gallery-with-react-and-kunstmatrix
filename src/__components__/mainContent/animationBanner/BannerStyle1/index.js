import React from "react";
import style from "./index.css";
import bannerimg from "./assets/banner.png";
import boy1 from "./assets/boy1.png";
import boy2 from "./assets/boy2.png";
import boy3 from "./assets/boy3.png";
import girl1 from "./assets/girl1.png";
import girl2 from "./assets/girl2.png";
import girl3 from "./assets/girl3.png";
import MultipleStyle from "classnames";
const Banner = () => {
  return (
    <div className={MultipleStyle(style.main_banner)}>
      <img className={style.boy_1} src={boy1} />
      <img className={style.boy_2} src={boy2} />
      <img className={style.boy_3} src={boy3} />
      <img className={style.girl_1} src={girl1} />
      <img className={style.girl_2} src={girl2} />
      <img className={style.girl_3} src={girl3} />
      <div className={style.boy_2} />
      <div className={style.boy_3} />
      <div className={style.girl_1} />
      <div className={style.girl_2} />
      <div className={style.girl_3} />
      <img className={style.main_banner_img} src={bannerimg} />
    </div>
  );
};
export default Banner;
