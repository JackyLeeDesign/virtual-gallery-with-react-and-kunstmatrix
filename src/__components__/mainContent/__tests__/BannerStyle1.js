import { configure } from "enzyme";
import renderer from "react-test-renderer";
import Adapter from "enzyme-adapter-react-16";
configure({ adapter: new Adapter() });
import React from "react";
import BannerStyle1 from "../animationBanner/BannerStyle1";
describe("BannerStyle1:UI Test", () => {
  it("UI test", () => {
    const component = renderer.create(<BannerStyle1 />);
    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
