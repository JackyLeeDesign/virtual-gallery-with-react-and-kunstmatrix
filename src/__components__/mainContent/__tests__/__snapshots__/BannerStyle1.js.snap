// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`BannerStyle1:UI Test UI test 1`] = `
<div
  className="main_banner"
>
  <img
    className="boy_1"
    src="boy1.png"
  />
  <img
    className="boy_2"
    src="boy2.png"
  />
  <img
    className="boy_3"
    src="boy3.png"
  />
  <img
    className="girl_1"
    src="girl1.png"
  />
  <img
    className="girl_2"
    src="girl2.png"
  />
  <img
    className="girl_3"
    src="girl3.png"
  />
  <div
    className="boy_2"
  />
  <div
    className="boy_3"
  />
  <div
    className="girl_1"
  />
  <div
    className="girl_2"
  />
  <div
    className="girl_3"
  />
  <img
    className="main_banner_img"
    src="banner.png"
  />
</div>
`;
